examples/stereo_depth_video.py

```diff
@@ -1,116 +1,299 @@
+
+    return pipeline, streams
+
+
+def get_lr_mesh(calibData):
+    M1 = np.array(calibData.getCameraIntrinsics(dai.CameraBoardSocket.LEFT, resolution[0], resolution[1]))
+    d1 = np.array(calibData.getDistortionCoefficients(dai.CameraBoardSocket.LEFT))
+    R1 = np.array(calibData.getStereoLeftRectificationRotation())
+    M2 = np.array(calibData.getCameraIntrinsics(dai.CameraBoardSocket.RIGHT, resolution[0], resolution[1]))
+    d2 = np.array(calibData.getDistortionCoefficients(dai.CameraBoardSocket.RIGHT))
+    R2 = np.array(calibData.getStereoRightRectificationRotation())
+    map_x_l, map_y_l = cv2.initUndistortRectifyMap(M1, d1, R1, M2, resolution, cv2.CV_32FC1)
+    map_x_r, map_y_r = cv2.initUndistortRectifyMap(M2, d2, R2, M2, resolution, cv2.CV_32FC1)
+
+    meshCellSize = 16
+    mesh_left = []
+    mesh_right = []
+
+    for y in range(map_x_l.shape[0] + 1):
+        if y % meshCellSize == 0:
+            row_left = []
+            row_right = []
+            for x in range(map_x_l.shape[1] + 1):
+                if x % meshCellSize == 0:
+                    if y == map_x_l.shape[0] and x == map_x_l.shape[1]:
+                        row_left.append(map_y_l[y - 1, x - 1])
+                        row_left.append(map_x_l[y - 1, x - 1])
+                        row_right.append(map_y_r[y - 1, x - 1])
+                        row_right.append(map_x_r[y - 1, x - 1])
+                    elif y == map_x_l.shape[0]:
+                        row_left.append(map_y_l[y - 1, x])
+                        row_left.append(map_x_l[y - 1, x])
+                        row_right.append(map_y_r[y - 1, x])
+                        row_right.append(map_x_r[y - 1, x])
+                    elif x == map_x_l.shape[1]:
+                        row_left.append(map_y_l[y, x - 1])
+                        row_left.append(map_x_l[y, x - 1])
+                        row_right.append(map_y_r[y, x - 1])
+                        row_right.append(map_x_r[y, x - 1])
+                    else:
+                        row_left.append(map_y_l[y, x])
+                        row_left.append(map_x_l[y, x])
+                        row_right.append(map_y_r[y, x])
+                        row_right.append(map_x_r[y, x])
+            if (map_x_l.shape[1] % meshCellSize) % 2 != 0:
+                row_left.append(0)
+                row_left.append(0)
+                row_right.append(0)
+                row_right.append(0)
+
+            mesh_left.append(row_left)
+            mesh_right.append(row_right)
+
+    mesh_left = np.array(mesh_left)
+    mesh_right = np.array(mesh_right)
+
+    return mesh_left, mesh_right
+
+
+def save_mesh_files(mesh_left, mesh_right, output_path):
+    print("Saving mesh to:", output_path)
+    mesh_left.tofile(output_path + "/left_mesh.calib")
+    mesh_right.tofile(output_path + "/right_mesh.calib")
+
+
+def get_disparity_frame(frame):
+    max_disp = 95
+    if extended:
+        max_disp *= 2
```

Comment:
outdated, use `stereo.initialConfig.getMaxDisparity()`
See stereo_depth_from_host example

Reply:
When using `stereo.initialConfig.getMaxDisparity()` an error is thrown saying the `StereoDepthConfig` has no attribute `getMaxDisparity`.
When using `stereo.outConfig.link(xout_stereo_config.input)` (the same way it's done in the stereo_depth_from_host example, an error is thrown saying `StereoDepth` node has no attribute `outConfig`.
So I'm using `stereo.getMaxDisparity()` and it seems to work:
 - 95.0 default
 - 190.0 extended disparity
 - 3040.0 subpixel disparity

Reply:
You are not using depthai library from `develop`